src/main/java/eu/dissco/orchestration/backend/service/SourceSystemService.java

```diff
@@ -108,6 +108,33 @@ private static boolean isEquals(SourceSystem sourceSystem, SourceSystem currentS
             currentSourceSystem.getLtcCollectionManagementSystem());
   }
 
+  @PostConstruct
```

Comment:
id put this at the top of the class

Reply:
Yeah I had it there until I did a reformat and it placed the static functions above it. Was in doubt what should come first but let's do the @PostConstruct